### ISSUE TITLE
Connecting via USB mutes PC audio

### ISSUE THREAD

**Issue by Hansukie** [Ryochan7/DS4Windows#850]
I just bought a new DS4 with the light bar on the touchpad. I have been previously using the first versions of the DS4 without the light on the touchpad and it's been working fine. 

However, the new DS4 doesn't relay the battery percentage correctly via USB or bluetooth, it always says it's at 0% no matter how long it's charged. Also, when connected via USB, all audio is muted on my PC. Disconnecting the DS4 restores audio functionality. 

Is there any way to fix this?

**Comment by mika-n** [comment]
Is the controller genuine Sony DS4 revision2 controller or some wanna-be compatible controller? 
Can you control the lightbar color and touchpad and gyro in DS4Windows app? If not then maybe the controller is some chinese fake copycat and not an official Sony controller? Often those chinese copies look pretty much the same as official gamepad, but lacks touchpad (other than click event) and gyro support.

Do you have PS4 console? Does the touchpad and gyro work there? It is often a good way to test if the controller really supports those features.

Try to run a special debug version of DS4Windows. It shows more details about the connected gamepad controller. It may tell us more about the controller and if DS4Windows somehow accidentally tries to connect an audio device of the controller.
https://github.com/Ryochan7/DS4Windows/wiki/Troubleshooting#detailed-debug-log-file-from-gamepad-connection-problems

**Comment by Hansukie** [comment]
It's an official Sony DS4. Unless the ones sold by Walmart are knockoffs, I believe it to be genuine. The touchpad functions correctly, tested using my PC to move the mouse cursor and on a PS4. The gyro and sixaxis functions correctly according to the display in DS4Windows.

I figured out why the audio gets muted when the controller is connected; Windows automatically switches all audio output and input over to the controller. Just going into Windows settings and switching over to the normal audio output device fixes this.

What I don't understand is why it does this with this new DS4 and not both of my old ones? Also, I did not know that the DS4 can output PC audio to itself through the headphone port on the controller. I thought this was limited to being connected to an actual PS4. It can do this when the DS4 is connected without DS4Windows running.
